Makefile

```diff
@@ -75,9 +76,13 @@ SHADER_OBJS = \
 	$(SRCDIR)/shaders/textured_pixelshader.inl \
 	$(SRCDIR)/shaders/untextured_pixelshader.inl
 
-DEBUG := y
 CFLAGS += -I$(SRCDIR) -I$(THIRDPARTYDIR)
-CXXFLAGS += -I$(SRCDIR) -I$(THIRDPARTYDIR)
+CXXFLAGS += -I$(SRCDIR) -I$(THIRDPARTYDIR) -DFPNG_NO_STDIO=1 -DFPNG_NO_SSE=1
```

Comment:
I haven't looked at the package but does SSE with fpng imply SSE2?

Reply:
fpng will use SSE 4.1+pclmul if available, but Xbox doesn't support it